config.yml

```diff
@@ -1,8 +1,8 @@
 settings:
-  accountName: "MyAccountName"
-  deploymentUid: "ss881269"
-  sitecorePrefix: "demo"
+  accountName: "HorizontalIntegration"
+  deploymentUid: "ss300127"
+  sitecorePrefix: "sitecore"
```

Comment:
This should really be reverted back to the demo ones, as you don't want anyone knowing your account details

Reply:
Thanks!